docs/index.html

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en-us">
     <head>
-	<meta name="generator" content="Hugo 0.98.0" />
+	<meta name="generator" content="Hugo 0.97.3" />
```

Comment:
Please don't downgrade the used Hugo version. It should either remain the same or be updated to the latest one.

Reply:
Oops, sorry. Fixed.